app/project_management/models/projects.py

```diff
@@ -100,6 +100,11 @@ class Meta:
     )
 
 
+    def __str__(self):
+
+        return self.name
+
+
     @property
     def percent_completed(self) -> str: # Auto-Calculate
         """ How much of the project is completed.
```

Comment:
#  add note on pr #251

- this needs to count tasks that are considered complete divided by number tasks times 100.